app/services/albums.js

```diff
@@ -12,9 +12,9 @@ const options = {
 
 exports.getAlbums = id => {
   if (id) options.uri = `${api_url}/${id}/photos`;
-  request_promise(options)
-    .then(logger.info)
+  return request_promise(options)
+    .then(res => res)
```

Comment:
unnecassary `.then()` you can use: 
```
return request_promise(options)
    .catch(error => {
      Promise.reject(externalApiError(error.message));
    });
```
because you are not doing anything in the `.then()`